### ISSUE TITLE
Feature: function to draw regions

### ISSUE THREAD

**Issue by gfs8966** [LakeErie-PreyFish-Acoustics/erieacoustics#25]
@HoldenJe - It should be possible to write a function in R that while connected to Echoview in a COM instance allows the creation of the epi and hypo line relative regions. Reach out to EV support team for help.

**Comment by mrdufour** [comment]
Yes, I've reached out to Echoview for help on these two enhancements.

**Comment by mrdufour** [comment]
Close.
7e1cd1290c495e515457af49e3f71333ff6a0891
